services/pkg/debuginfod/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM voidlinux/voidlinux-musl:20201117RC01
+RUN xbps-install -Sy debuginfod && \
+        rm -rf /var/cache/xbps
+WORKDIR /debuginfod
+ENTRYPOINT ["/usr/bin/debuginfod"]
+CMD ["-F", "-d", "/debuginfod/db.sqlite", "-I", ".*-dbg-.*", "/binpkgs"]
```

Comment:
I have `debuginfod -Z.xbps <path>` as the command here; I believe you use -F only if you have bare ELFs and DWARFs (I swear this isn't an RPG)